sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2Exec.scala

```diff
@@ -350,15 +350,15 @@ trait BatchWriteHelper {
 /**
  * The base physical plan for writing data into data source v2.
  */
-trait V2TableWriteExec extends UnaryExecNode {
+trait V2TableWriteExec extends V2CommandExec with UnaryExecNode {
```

Comment:
The physical plan still shows in the Spark UI because `UnaryExecNode` overrides `V2CommandExec.children` right?

Reply:
yes. The fact that the data is inserted means that the child does get overridden, and the UI should look fine as well